### ISSUE TITLE
Model retraining with market 1501 produces very low mAP

### ISSUE THREAD

**Issue by voqtuyen** [VisualComputingInstitute/triplet-reid#63]
I have retrained the code with the same config and on Market 1501 dataset but after evaluation, the result i get is very low mAP:

> python train.py \
>     --train_set data/market1501_train.csv \
>     --model_name mobilenet_v1_1_224 \
>     --image_root $IMAGE_ROOT \
>     --initial_checkpoint $INIT_CHECKPT \
>     --experiment_root $EXP_ROOT \
>     --flip_augment \
>     --crop_augment \
>     --embedding_dim 128 \
>     --batch_p 18 \
>     --batch_k 4 \
>     --pre_crop_height 288 --pre_crop_width 144 \
>     --net_input_height 256 --net_input_width 128 \
>     --margin soft \
>     --metric euclidean \
>     --loss batch_hard \
>     --learning_rate 3e-4 \
>     --train_iterations 25000 \
>     --decay_start_iteration 15000 \

> mAP: 1.43% | top-1: 3.30% top-2: 4.84% | top-5: 8.14% | top-10: 11.64%

Any idea on how to fix it!

**Comment by voqtuyen** [comment]
Thanks, i will continue to check and give you any more information
